python/cudf/cudf/core/dtypes.py

```diff
@@ -268,6 +268,15 @@ def deserialize(cls, header: dict, frames: list):
 
         return cls(element_type=element_type)
 
+    @property
+    def nesting_levels(self):
+        leaf_typ = self.element_type
```

Comment:
Can we add a succinct comment to establish exactly what this property is returning? Maybe something like: "Returns an integer specifying the level of nesting present in this ListDtype instance"

Reply:
Added the docs